srcpkgs/shotgun/template

```diff
@@ -0,0 +1,25 @@
+# Template file for 'shotgun'
+pkgname=shotgun
+version=2.2.0
+revision=1
+archs="x86_64 i686 x86_64-musl"
```

Comment:
Is it failing because arm or failing because cross-build? If its the latter, then it should be marked `nocross` instead

Reply:
The following patch fixes the cast so it works on all archs.

    --- src/xwrap.rs.orig
    +++ src/xwrap.rs
    @@ -248,7 +248,7 @@
             let mut y = 0;
             let mut w = 0;
             let mut h = 0;
    -        xlib::XParseGeometry(g.as_ptr() as *const i8, &mut x, &mut y, &mut w, &mut h);
    +        xlib::XParseGeometry(g.as_ptr() as *const raw::c_char, &mut x, &mut y, &mut w, &mut h);
     
             util::Rect {
                 x: x,

Drop it in `srcpkgs/shotgun/patches`